nixos/modules/services/mail/public-inbox.nix

```diff
@@ -126,28 +117,9 @@ let
           # Not removing @timer because git upload-pack needs it.
         ];
         SystemCallArchitectures = "native";
-
-        # The following options are redundant when confinement is enabled
-        RootDirectory = "/var/empty";
-        TemporaryFileSystem = "/";
-        PrivateMounts = true;
-        MountAPIVFS = true;
-        PrivateDevices = true;
-        PrivateTmp = true;
-        PrivateUsers = true;
-        ProtectControlGroups = true;
-        ProtectKernelModules = true;
-        ProtectKernelTunables = true;
       };
       confinement = {
-        # Until we agree upon doing it directly here in NixOS
-        # https://github.com/NixOS/nixpkgs/pull/104457#issuecomment-1115768447
-        # let the user choose to enable the confinement with:
```

Comment:
Are there now other modules that use confinement by default?

Reply:
@alyssais, since https://github.com/NixOS/nixpkgs/pull/289593 has been merged last May, those modules enable `confinement` by default:
- [radicle](https://github.com/NixOS/nixpkgs/blob/ab6998e83282a01827f660e5e25ea526ac559a50/nixos/modules/services/misc/radicle.nix#L103) (co-maintained by me)
- [cryptpad](https://github.com/NixOS/nixpkgs/blob/ab6998e83282a01827f660e5e25ea526ac559a50/nixos/modules/services/web-apps/cryptpad.nix#L209) (not co-maintained by me)

Reply:
Thanks! OOC, in https://github.com/NixOS/nixpkgs/pull/234641 (which maybe I should finish), I noted that DynamicUser was not compatible with confinement.  Do you know if that's changed?

Reply:
@alyssais, yes in https://github.com/NixOS/nixpkgs/pull/289593 we made `confinement` compatible with `ProtectSystem=strict` (and thus with `DynamicUser=1` too). I'm following https://github.com/NixOS/nixpkgs/pull/234641 and will review it, should you have time to finish it.

Reply:
Great, thanks :)